Registry/registry.clm

```diff
@@ -15,12 +15,23 @@ state   real    WTP            i4j      misc      1          Z     rd=(interp_ma
 state   real    H2OSNO         i4j      misc      1          Z     rd=(interp_mask_field:lu_index,iswater)u=(copy_fcnm)           "H2OSNO"   "SUBGRID SNOW WATER EQUIVALENT" "kg m-2"
 state   real    T_GRND         i4j      misc      1          Z     rhd=(interp_mask_field:lu_index,iswater)u=(copy_fcnm)           "T_GRND"   "SUBGRID GROUND TEMPERATURE"     "K"
 state   real    T_VEG          i4j      misc      1          Z     rhd=(interp_mask_field:lu_index,iswater)u=(copy_fcnm)           "T_VEG"    "SUBGRID VEGETATION TEMPERATURE" "K"
+state   real    T_VEG24        i4j      misc      1          Z     rhd=(interp_mask_field:lu_index,iswater)u=(copy_fcnm)          "T_VEG24"  "24H SUBGRID VEGETATION TEMPERATURE" "K"
+state   real    T_VEG240       i4j      misc      1          Z     rhd=(interp_mask_field:lu_index,iswater)u=(copy_fcnm)          "T_VEG240" "10D SUBGRID VEGETATION TEMPERATURE" "K"
+state   real    FSUN           i4j      misc      1          Z     rhd=(interp_mask_field:lu_index,iswater)u=(copy_fcnm)          "FSUN"     "SUBGRID VEGETATION RADIATION" "W m-2"
+state   real    FSUN24         i4j      misc      1          Z     rhd=(interp_mask_field:lu_index,iswater)u=(copy_fcnm)          "FSUN24"   "24H SUBGRID VEGETATION RADIATION" "W m-2"
+state   real    FSUN240        i4j      misc      1          Z     rhd=(interp_mask_field:lu_index,iswater)u=(copy_fcnm)          "FSUN240"  "10D SUBGRID VEGETATION RADIATION" "W m-2"
+state   real    FSD24          i4j      misc      1          Z     rhd=(interp_mask_field:lu_index,iswater)u=(copy_fcnm)          "FSD24"    "24H DIRECT SOLAR RADIATION" "W m-2"
+state   real    FSD240         i4j      misc      1          Z     rhd=(interp_mask_field:lu_index,iswater)u=(copy_fcnm)          "FSD240"   "10D DIRECT SOLAR RADIATION" "W m-2"
+state   real    FSI24          i4j      misc      1          Z     rhd=(interp_mask_field:lu_index,iswater)u=(copy_fcnm)          "FSI24"    "24H INDIRECT SOLAR RADIATION" "W m-2"
+state   real    FSI240         i4j      misc      1          Z     rhd=(interp_mask_field:lu_index,iswater)u=(copy_fcnm)          "FSI240"   "10D INDIRECT SOLAR RADIATION" "W m-2"
+state   real    LAIP           i4j      misc      1          Z     rhd=(interp_mask_field:lu_index,iswater)u=(copy_fcnm)          "LAIP"     "PRIOR TIMESTEP LEAF AREA INDEX" "area/area"
```

Comment:
@brigaudet Can you see if you can package these new arrays (with a new PR)? Thanks.